gdk-pixbuf/Cargo.toml

```diff
@@ -9,9 +9,7 @@ documentation = "https://gtk-rs.org/gtk-rs-core/stable/latest/docs/gdk_pixbuf/"
 version = "0.19.0"
 description = "Rust bindings for the GdkPixbuf library"
 repository = "https://github.com/gtk-rs/gtk-rs-core"
-exclude = [
-    "gir-files/*",
-]
+exclude = ["tests", "Gir.toml", "docs.md", "versions.txt"]
```

Comment:
Why do you want to exclude the tests (they're useful for `cargo test`) and `versions.txt` (useful so people know what version of everything this is based on so they can reproduce it).

Reply:
crates.io is not only used for pulling in dependencies but also as a source for e.g. Linux distro packagers (who need these two things, for example). Maybe that's the misconception here?

Reply:
Yes, that is where I was wrong. I did not know that.

So packagers fetch the crates even for libraries such as gtk-rs and run tests on them or check the versions file? Is there anywhere, where I can read more on this topic?

Reply:
No problem, I'll close both PRs. Thank you for letting me know

Reply:
No problem and thanks for trying to make these crates smaller :)